src/main/webapp/chart.html

```diff
@@ -4,34 +4,39 @@
     <title>A New Chart</title>
     <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
     <script type="text/javascript">
-	    //loads the corecharts package, defined in loader.js script above
-	    google.charts.load('current', {packages: ['corechart']});
-	    //once the corecharts package is loades, call drawChart
-	    google.charts.setOnLoadCallback(drawChart);
-	    //actually draws the chart
-	    function drawChart(){
-	    	//creates new datatable instance. each column has a datatype, label (optional) and id (optional)
-	    	var book_data = new google.visualization.DataTable();
+        google.charts.load('current', {packages: ['corechart']});
+        google.charts.setOnLoadCallback(drawChart);
+        function drawChart(){
+          fetch("/bookchart")
+              .then((response) => {
+                return response.json();
+              })
+              .then((bookJson) => {
+                var bookData = new google.visualization.DataTable();
+                //define columns for the DataTable instance
+                bookData.addColumn('string', 'Book Title');
+                bookData.addColumn('number', 'Rating');
 
-	    	//creating two columns for datatable
-	    	book_data.addColumn('string', 'Country');
-	    	book_data.addColumn('number', 'Population (million)'); 
+                for (i = 0; i < bookJson.length; i++) {
+                  bookRow = [];
+                  var title = bookJson[i].title;
+                  var ratings = bookJson[i].rating;
+                  bookRow.push(title, ratings);
 
-	    	//adding data for datatable
-	    	book_data.addRows([
-	    		["United States", 300],
-	    		["Canada", 37],
-	    		["Mexico", 129],
-	    	]); 
-
-	    	//create new instance of chart and draw it 
-	    	var chart = new google.visualization.GeoChart(document.getElementById('book_chart')); 
-	    	chart.draw(book_data); 
-	    }
+                  bookData.addRow(bookRow);
+               }
+               var chartOptions = {
+                                 width: 800,
+                                 height: 400
+                   };
```

Comment:
You should move lines 29-31 back a bit to align them and to keep things consistent.